unity_buy_android/unity_buy_plugin/src/main/java/com/shopify/unity/buy/UnityAndroidPayFragment.java

```diff
@@ -0,0 +1,39 @@
+package com.shopify.unity.buy;
+
+import android.app.Fragment;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+
+import com.shopify.buy3.pay.PayCart;
+
+public class UnityAndroidPayFragment extends Fragment {
+    private PayCart cart;
+    private String unityDelegateObjectName;
+
+    private static String EXTRA_UNITY_DELEGATE_OBJECT_NAME = "unityDelegateObjectName";
```

Comment:
Yeah, usually we call this in this way to indicate that this key belongs to parsing extras.